src/kpi/statistics/lib/serialize.ts

```diff
@@ -0,0 +1,38 @@
+import { Intervals } from '.';
+
+export function serialize(
```

Comment:
Take a result of a MongoDB pipeline and transform it into a JS object that can be served as response to the HTTP request.

Reply:
Sadly, it's not. If the prepared pipeline was directly executed, it would be no problem. But since we have the clear intention to add additional stages to the pipeline, the resulting type changes.